maven/build.go

```diff
@@ -184,3 +204,12 @@ func contains(strings []string, stringsSearchedAfter []string) bool {
 	}
 	return false
 }
+
+// Added to ensure formatting of mvnw file is unix compatible
+func replaceLineEndings(d []byte) []byte {
+	// replace CRLF with LF
+	d = bytes.Replace(d, []byte{13, 10}, []byte{10}, -1)
```

Comment:
I would suggest `bytes.ReplaceAll` instead.

https://pkg.go.dev/bytes#ReplaceAll

It does the same thing, but it reads cleaner. You don't have to think what the `-1` means here.